lib/hijack/error.js

```diff
@@ -58,12 +58,17 @@ export function TrackUnhandledRejections () {
   process.on('unhandledRejection', function (reason) {
     // skip errors with `_skipKadira` flag
     if (
+      reason &&
       reason._skipKadira ||
       !Kadira.options.enableErrorTracking
     ) {
       return;
     }
 
+    if (!reason) {
+      reason = new Error('unhandledRejection: reason undefined');
```

Comment:
I don't think it is common knowledge that this is called `reason` - users might incorrectly think it is referring to a `reason` variable in app or package code. Changing it to `unhandledRejection: undefined` might make it more clear.

The UI indicates when an error is from an unhandled promise rejection, so the message doesn't need to include it, though it doesn't hurt to have it.